apps/cowswap-frontend/src/modules/ordersTable/utils/getOrderParams.ts

```diff
@@ -43,7 +45,7 @@ export function getOrderParams(
     partiallyFillable: order.partiallyFillable,
     sellAmount,
     balance,
-    allowance,
+    allowance: getBiggerAmount(allowance, permitAmount),
```

Comment:
I take allowance value from blockchain OR permit value from order appData, depending on what is bigger